ash/src/device.rs

```diff
@@ -36,6 +38,243 @@ impl Device {
     }
 }
 
+/// Vulkan core 1.3
+#[allow(non_camel_case_types)]
+impl Device {
+    pub fn fp_v1_3(&self) -> &vk::DeviceFnV1_3 {
+        &self.device_fn_1_3
+    }
+
+    /// <https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkCreatePrivateDataSlot.html>
```

Comment:
should the doc point to version 1.3 ?

Reply:
I guess I'll correct that in the generator in #563 and find-replace it in manual code. The links are still valid though, they just redirect. Don't ask me why these nice isolated pages are behind a `-extensions` URL anyway...